pkgs/applications/office/ib/controller/default.nix

```diff
@@ -155,6 +155,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Automation Controller for the Trader Work Station of Interactive Brokers";
+    broken = true;
```

Comment:
Instead, please add comment here and there!

Reply:
Sounds good -- moved in a4afd7b04